install/collecting_ncn_mac_addresses.md

```diff
@@ -72,7 +72,22 @@ For help with either of those, see [LiveCD Setup](bootstrap_livecd_remote_iso.md
     pit# sleep 10
     pit# grep -oP "($mtoken|$stoken|$wtoken)" /etc/dnsmasq.d/statics.conf | sort -u | xargs -t -i ipmitool -I lanplus -U $USERNAME -E -H {} power on
     ```
-4. Now wait for the nodes to netboot. You can follow them with `conman -j ncn-*id*-mgmt` (use `conman -q` to see ). This takes less than 3 minutes, speed depends on how quickly your nodes POST.
+4. Now wait for the nodes to netboot. You can follow them with `conman -m ncn-*id*-mgmt` (use `conman -q` to see the list of nodes). This takes less than 3 minutes, speed depends on how quickly your nodes POST.  To exit the console session, use the command `&.` and to see a conman help screen for all supported escape sequences use `&?`.
```

Comment:
The use of "ncn-*id*-mgmt" is not clear.  We do not refer to "id" anywhere else in the documentation.   And our documentation style tries to avoid calling the node "m003" when it is really the hostname "ncn-m003".  The hostnames for the management nodes are of the form ncn-AXXX, where A is one of (m,s,w) and XXX is a 3 digit number with zero padding.  Don't need to explain this all here.

What I think you intend is that the hostname of the management node have the suffix "-mgmt" added to it.



Reorder to have this before the 2nd one
Use conman -q
Follow with conman -j

I notice you changed from conman -j to conman -m.  That makes sense for this use as read-only.  But you might want to highlight that this is readonly and mention that if there are problems and interaction with the node console is needed that they should switch from conman -m to conman -j.